From 18ee835acd542161dac63e730f862f9598b22b48 Mon Sep 17 00:00:00 2001
From: brendaan07 <brendaaaan07@gmail.com>
Date: Sat, 31 May 2025 16:59:37 +0800
Subject: [PATCH 2/8] updatetabiconcolor

---
 app/_layout.jsx | 16 +++++++++++-----
 1 file changed, 11 insertions(+), 5 deletions(-)

diff --git a/app/_layout.jsx b/app/_layout.jsx
index 5bd49b86..7c420ec3 100644
--- a/app/_layout.jsx
+++ b/app/_layout.jsx
@@ -5,13 +5,19 @@ import streaksIcon from "../assets/StreaksIcon.png";
 
 const _layout = () => {
   return (
-    <Tabs> //creates tab navigation 
+    <Tabs //creates tab navigation 
+        screenOptions={{
+            headerShown: false,
+            tabBarActiveTintColor: "black",
+            tabBarInactiveTintColor: "gray",
+        }}
+    > 
         <Tabs.Screen //edit tab screen
             name="index"
             options={{ 
                 title: "Home",
-                tabBarIcon: () => (
-                    <Image source={homeIcon} style={{ width: 24, height: 24 }} />
+                tabBarIcon: ({ color }) => (
+                    <Image source={homeIcon} style={{ width: 24, height: 24, tintColor: color }} />
                 )
             }}
         />
@@ -19,8 +25,8 @@ const _layout = () => {
             name="streaks"
             options={{ 
                 title: "Streaks",
-                tabBarIcon: () => (
-                    <Image source={streaksIcon} style={{ width: 24, height: 24 }} />
+                tabBarIcon: ( {color}) => (
+                    <Image source={streaksIcon} style={{ width: 24, height: 24, tintColor: color}} />
                 )
             }}
         />
-- 
2.47.1.windows.1

